server/index.js

```diff
@@ -1,25 +1,28 @@
-/* eslint-disable no-console */
 const path = require('path');
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
-const Restaurant = require('../database/Restaurant.js');
+const cors = require('cors');
+const db = require('../database/index');
+const { searchById } = require('../database/Restaurant');
 
 const app = express();
-
-app.use(express.static(path.join(__dirname, '../client/dist')));
-
+app.use(cors());
 app.use(morgan('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(
+  '/r/:restaurants',
+  express.static(path.join(__dirname, '../client/dist')),
```

Comment:
This line (line 16) throws an error and prevents the server from running. Take off the comma from the end of this line.